app/views/includes/organisation-overview.html

```diff
@@ -1,3 +1,14 @@
+<table>
+    <tr>
+        <td>
+        {% ifAsync ((organisationLevel === 'region') or (organisationLevel === 'ldu') or (organisationLevel === 'team')) %}
+        <a href="/probation/{{ organisationLevel }}/{{ linkId }}/{{ screen }}/reductions-csv" class="button reduction-export" style="float: right; margin-left:10px; margin-bottom: 10px">Export Reductions</a>
+        {% endif %}
+        <a href="/probation/{{ organisationLevel }}/{{ linkId }}/{{ screen }}/caseload-csv" class="button sln-export" style="float: right; margin-bottom: 10px; ">Export Caseload</a>
```

Comment:
Remove reduction-export and sln-export from class attributes if these are no longer specified. Same in export-caseload.html

Reply:
reduction-export and sln-export are used to tell the difference between the buttons on the e2e tests to make sure both are displayed